core/services/ocr2/plugins/vault/plugin.go

```diff
@@ -35,10 +35,10 @@ const (
 	defaultMaxIdentifierOwnerLengthBytes     = 64
 	defaultMaxIdentifierNamespaceLengthBytes = 64
 
-	defaultLimitsMaxQueryLength                          = 1024 // 1KB
-	defaultLimitsMaxObservationLength                    = 1024 // 1KB
-	defaultLimitsMaxReportsPlusPrecursorLength           = 1024 // 1KB
-	defaultLimitsMaxReportLength                         = 1024 // 1KB
+	defaultLimitsMaxQueryLength                          = 1024   // 1KB
+	defaultLimitsMaxObservationLength                    = 102400 // 100KB
+	defaultLimitsMaxReportsPlusPrecursorLength           = 1024   // 1KB
```

Comment:
I would expect this to need to be bumped too if your hitting the max report limit